tests/kernel/pipe/pipe/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+tests:
+  kernel.pipe:
+      tags: kernel
+  kernel.pipe.userspace:
```

Comment:
we do not need the filtering and extra configs, those are already defined in qemu_x86 so we should be fine with the default. Otherwise we are going to run the test twice without any difference